nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/resources/OSGI-INF/copy-schema-test-contrib.xml

```diff
@@ -1,6 +1,16 @@
 <?xml version="1.0"?>
-<component name="org.nuxeo.ecm.automation.core.operations.document.copyschema.test">
+<component
+  name="org.nuxeo.ecm.automation.core.operations.document.copyschema.test">
```

Comment:
No need to reindent this. Check you IDE line length for XML files, it should be the same as Java.